src/current/v23.1/backup-architecture.md

```diff
@@ -93,6 +93,40 @@ The backup metadata files describe everything a backup contains. That is, all th
+
+### Job coordination and export of locality-aware backups
+
+When you create a [locality-aware backup](take-and-restore-locality-aware-backups.html) job, any node in the cluster can [claim the backup job](#job-creation-phase). A successful locality-aware backup job requires that each node in the cluster has access to each storage location. This is because any node in the cluster can claim the job and become the coordinator node. Once each node informs the coordinator node that it has completed exporting the row data, the coordinator will start to write metadata, which involves writing to each locality bucket a partial manifest recording what row data was written to that [storage bucket](use-cloud-storage.html).
+
+Every node involved in the backup is responsible for backing up the ranges for which it was the [leaseholder](architecture/replication-layer.html#leases) at the time the coordinator planned the [distributed backup flow](backup-architecture.html#resolution-phase). The locality of the node ([configured at node startup](cockroach-start.html#locality)) exporting the row data determines where the backups files will be placed in a locality-aware backup.
+
+The node exporting the row data, and the leaseholder of the range being backed up, are usually the same. However, these nodes can differ when lease transfers have occurred during the [execution](#export-phase) of the backup. In this case, the leaseholder node returns the files to the node exporting the backup data (usually a local transfer), which then writes the file to the external storage location with a locality that matches its own localities (with an overall preference for more specific values in the locality hierarchy). If there is no match, the `default` locality is used.
+
+For example, in the following diagram there is a three-node cluster split across three regions. The leaseholders write the ranges to be backed up to the external storage in the same region. As **Nodes 1** and **3** complete their work, they send updates to the coordinator node (**Node 2**). The coordinator will then [write the partial manifest files](#metadata-writing-phase) containing metadata about the backup work completed on each external storage location, which is stored with the backup SST files written to that storage location.
+
+During a [restore](restore.html) job, the job creation statement will need access to each of the storage locations to read the metadata files in order to complete a successful restore.
+
+<img src="{{ 'images/v23.1/locality-aware-backups.png' | relative_url }}" alt="How a locality-aware backup writes to storage buckets in each region" style="border:0px solid #eee;max-width:100%" />
+
+### Job coordination using the `EXECUTION LOCALITY` option
+
```

Comment:
This has been moved from the locality restricted page to the backup architecture page.